README.md

```diff
@@ -2,16 +2,18 @@
 
 ## How to install ?
 Clone the project and install the dependencies in the `src` folder by running
-```npm install```
+```
```

Comment:
While we're at it, we can make this "```console".

Reply:
wdym by this?

Reply:
https://stackoverflow.com/questions/20303826/how-to-highlight-bash-shell-commands-in-markdown